hub/instruction_handling/create/generalities/unexceptional.tex

```diff
@@ -60,23 +60,62 @@
 		\[
 			\left\{ \begin{array}{lcl}
 				% \accRlpAddrFlag               _{i + \createFirstCreatorAccountRowOffset} & = & \rOne              \\
-				\accRlpAddrRecipe             _{i + \createFirstCreatorAccountRowOffset} & = & \locRlpAddrRecipe  \\
-				\accRlpAddrDepAddrHi          _{i + \createFirstCreatorAccountRowOffset} & = & \relevantValue     \\
-				\accRlpAddrDepAddrLo          _{i + \createFirstCreatorAccountRowOffset} & = & \relevantValue     \\
-				\accRlpAddrSaltHi             _{i + \createFirstCreatorAccountRowOffset} & = & \locSaltHi         \\
-				\accRlpAddrSaltLo             _{i + \createFirstCreatorAccountRowOffset} & = & \locSaltLo         \\
-				\accRlpAddrKecHi              _{i + \createFirstCreatorAccountRowOffset} & = & \locInitCodeHashHi \\
-				\accRlpAddrKecLo              _{i + \createFirstCreatorAccountRowOffset} & = & \locInitCodeHashLo \\
+				\accRlpAddrRecipe    _{i + \createFirstCreatorAccountRowOffset} & = & \locRlpAddrRecipe  \\
+				\accRlpAddrDepAddrHi _{i + \createFirstCreatorAccountRowOffset} & = & \relevantValue     \\
+				\accRlpAddrDepAddrLo _{i + \createFirstCreatorAccountRowOffset} & = & \relevantValue     \\
+				\accRlpAddrSaltHi    _{i + \createFirstCreatorAccountRowOffset} & = & \locSaltHi         \\
+				\accRlpAddrSaltLo    _{i + \createFirstCreatorAccountRowOffset} & = & \locSaltLo         \\
+				\accRlpAddrKecHi     _{i + \createFirstCreatorAccountRowOffset} & = & \locInitCodeHashHi \\
+				\accRlpAddrKecLo     _{i + \createFirstCreatorAccountRowOffset} & = & \locInitCodeHashLo \\
 			\end{array} \right.
 		\]
 		where we set
 		\[
-			\locRlpAddrRecipe := 
-			\left[ \begin{array}{lrcl}
-				+ & \locIsCreate         & \!\!\! \cdot \!\!\! & \addressCreationRecipeConstantStd \\
-				+ & \locIsCreateTwo      & \!\!\! \cdot \!\!\! & \addressCreationRecipeConstantTwo \\
-			\end{array} \right]
+			\left\{ \begin{array}{lcl}
+				\locRlpAddrRecipe & \define &
+				\left[ \begin{array}{lrcl}
+					+ & \locIsCreate         & \!\!\! \cdot \!\!\! & \addressCreationRecipeConstantStd \\
+					+ & \locIsCreateTwo      & \!\!\! \cdot \!\!\! & \addressCreationRecipeConstantTwo \\
+				\end{array} \right] \vspace{2mm} \\
+				\locInitCodeHashHi & \define & \locIsCreateTwo \cdot
+				\left[ \begin{array}{rcl}
+					\locTriggerHashInfo       & \cdot & \locHashInfoKeccakHi \\
```

Comment:
![Screenshot 2024-12-03 at 18 24 33](https://github.com/user-attachments/assets/ce6cd37f-0a66-404f-b01a-6b9a5a761105)
this expression seems to be not well-formed.

Reply:
Yes, it's missing `+` signs, will fix.

Reply:
Done